### DISCUSSION TITLE
Keyframe handling

### DISCUSSION THREAD

**Discussion by zigaturk** [mifi/lossless-cut#701] [Ideas]
When a cut is made not at keyframe, why not insert, at the beginning, as a keyfame, a render of the video at that point. It could then remain static until next keyframe. This way beginnings of videos would not be without picture.

An even better solution would be to render that second or so and then simply copy the rest of the video. The tool would still be fast.

Also (this is a bug report) I often manually set cut points at keyframes yet it is often cut as if the cutpoint was just after the keyframe.

**Comment by mifi** [comment]
I’m not sure how to insert a frame into a video stream losslessly actually. If someone has any advice please share.

Cutting: yea it’s a known issue, sometimes ffmpeg cuts at the previous frame. Im not sure why. We have been discussing previously maybe setting the cut time a bit later than the desired cut point
